core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -520,7 +520,7 @@ private String transformOMAScore( String omaScore) {
         } else if( omaScore.equalsIgnoreCase("N") || omaScore.equalsIgnoreCase("neutral")) {
             return "N";
         } else if( omaScore.equalsIgnoreCase("[sent]")) {
-            return ExtendedMutationUtil.NOT_AVAILABLE; // TODO temp workaround for invalid sent values
+            return "NA"; // TODO temp workaround for invalid sent values
```

Comment:
@pieterlukasse `getma.org` is dead, ideally we should replace them with the correct "mutationassessor.org" links. Currently, we are dealing with this dead links in the frontend by string replacing... But eventually, we hope to move mutation assessor related code from portal backend to genome nexus instead.